tests/foreman/cli/test_errata.py

```diff
@@ -1125,81 +1128,69 @@ def test_positive_check_errata_dates(module_entitlement_manifest_org, module_tar
 
 
-@pytest.fixture(scope='module')
-def rh_repo_module_manifest(module_entitlement_manifest_org, module_target_sat):
-    """Use module manifest org, creates RH tools repo, syncs and returns RH repo."""
-    # enable rhel repo and return its ID
-    rh_repo_id = module_target_sat.api_factory.enable_rhrepo_and_fetchid(
-        basearch=DEFAULT_ARCHITECTURE,
-        org_id=module_entitlement_manifest_org.id,
-        product=PRDS['rhel'],
-        repo=REPOS['rhst7']['name'],
-        reposet=REPOSET['rhst7'],
-        releasever=None,
-    )
-    # Sync step because repo is not synced by default
-    rh_repo = module_target_sat.api.Repository(id=rh_repo_id).read()
-    rh_repo.sync()
-    return rh_repo
-
-
-"""Section for tests using RHEL7.7 Content Host.
+"""Section for tests using RHEL Content Host.
    The applicability tests using Default Content View are related to the introduction of Pulp3.
    """
 
 
 @pytest.fixture(scope='module')
-def new_module_ak(module_entitlement_manifest_org, rh_repo_module_manifest, default_lce):
-    new_module_ak = entities.ActivationKey(
-        content_view=module_entitlement_manifest_org.default_content_view,
-        environment=entities.LifecycleEnvironment(id=module_entitlement_manifest_org.library.id),
-        organization=module_entitlement_manifest_org,
+def new_module_ak(
+    module_sca_manifest_org, rh_repo_module_manifest, module_lce_library, module_target_sat
+):
+    new_module_ak = module_target_sat.api.ActivationKey(
+        environment=module_lce_library,
+        organization=module_sca_manifest_org,
     ).create()
     # Ensure tools repo is enabled in the activation key
     new_module_ak.content_override(
-        data={'content_overrides': [{'content_label': REPOS['rhst7']['id'], 'value': '1'}]}
-    )
-    # Fetch available subscriptions
-    subs = entities.Subscription(organization=module_entitlement_manifest_org).search(
-        query={'search': f'{DEFAULT_SUBSCRIPTION_NAME}'}
+        data={'content_overrides': [{'content_label': REPOS['rhsclient8']['id'], 'value': '1'}]}
     )
-    assert subs
-    # Add default subscription to activation key
-    new_module_ak.add_subscriptions(data={'subscription_id': subs[0].id})
-    return new_module_ak
+    return new_module_ak.read()
 
 
 @pytest.fixture
 def errata_host(
     target_sat,
-    rhel8_contenthost,
-    module_entitlement_manifest_org,
+    rhel_contenthost,
+    module_sca_manifest_org,
     module_lce_library,
     new_module_ak,
 ):
-    """A RHEL8 Content Host that has applicable errata and registered to Library, using Global Registration."""
-    org = module_entitlement_manifest_org
+    """A RHEL Content Host that has applicable errata and registered to Library, using Global Registration."""
+    """ To use specific RHEL version(s), parameterize any test importing this fixture like so:
+            @pytest.mark.rhel_ver_match('[7, 8, ...]')
+    """
```

Comment:
```suggestion
    """A RHEL Content Host that has applicable errata and registered to Library, using Global Registration
         To use specific RHEL version(s), parameterize any test importing this fixture like so:
            @pytest.mark.rhel_ver_match('[7, 8, ...]')
    """
```

Reply:
Looks like this was a pre-commit format, fixing